packages/flutter/lib/src/widgets/ticker_provider.dart

```diff
@@ -62,30 +62,87 @@ class TickerMode extends StatelessWidget {
     return widget?.enabled ?? true;
   }
 
+  /// Obtains a [ValueNotifier] from the [TickerMode] surrounding the `context`,
+  /// which indicates whether tickers are enabled in the given subtree.
+  ///
+  /// When tickers are enabled or disabled, the notifier notifies its listeners.
+  ///
+  /// While the [ValueNotifier] is stable for the lifetime of the surrounding
+  /// [TickerMode], calling this method does not establish a dependency between
+  /// the `context` and the [TickerMode]. When the [Widget] owning the `context`
+  /// is moved to a new location in the tree where it may have a different
+  /// [TickerMode] ancestor, it is up to the [Widget] to obtain a new
+  /// [ValueNotifier] from the new ancestor [TickerMode] by calling this method
+  /// again. [StatefulWidget]s can, for example, do this in [State.activate].
```

Comment:
Express this more prescriptively: if the widget is moved, it must unsubscribe from the old notifier and subscribe to any new one that might be available, the correct place to do this is state.deactivate/activate.